apps/judicial-system/backend/src/app/modules/case/case.controller.ts

```diff
@@ -359,6 +360,16 @@ export class CaseController {
           }
         }
         break
+      case CaseTransition.WITHDRAW_APPEAL:
+        // if the case has not been received by the court of appeals, the appeal is withdrawn
+        // without a ruling decision
+        if (
+          !theCase.appealRulingDecision &&
+          theCase.appealState === CaseAppealState.RECEIVED
+        ) {
```

Comment:
The reason I also checked the appealRulingDecision is because I didn't want to overwrite a decision in case the COA had already put one down, it felt weird to me to change their decision. I'm not sure it's a case that would ever come up though so might not really matter.